### PR TITLE
Added documentation for required SAP downloads

### PR DESCRIPTION
**Pull request by tniek**

### CODE REVIEW

File: downloads.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Required SAP packages
+This document contains a "laundry list" of SAP packages ("bits") used by the automated deployment templates. Before you can run the Terraform modules and Ansible playbooks/roles, you'll need to:
```
Comment (abhinababasu) [inline downloads.md]: I don't think Laundry list sounds right.

Comment (abhinababasu) [reply downloads.md]: This document contains a list of SAP packages used by the automated deployment templates. Before you can run the Terraform modules and Ansible playbooks/roles, you will need to do the following steps.

File: downloads.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* use the table below to identify which bits are required for your scenario;
```
Comment (abhinababasu) [inline downloads.md]: I thought you wanted to provide some sort of script to manage the downloaded  bits? E.g. put them into different blobs? Also having the az cli commands to upload to a blob is good.

File: downloads.md (hunk 1 above)
Comment (tniek) [reply downloads.md]: @abhinababasu I'm fine with your version.

File: downloads.md (hunk 2 above)
Comment (tniek) [reply downloads.md]: @abhinababasu That will be the long-term goal; however, there are a few legal and technical challenges (i.e. download of bits for non-SAP employees is only possible through a proprietary download manager). Just wanted to add these instructions in the meantime, so our users know where to get the bits from.

**Pull request merged**